Build with --no-default-features fail: could not find `sink` in `futures_util`

## Current Behavior
Good day. If current Actix master branch build with `--no-default-features` flag, an error appears:
```
error[E0432]: unresolved import `futures_util::sink`
  --> src/io.rs:10:19
   |
10 | use futures_util::sink::Sink;
   |                   ^^^^ could not find `sink` in `futures_util`
```

## Possible Solution
To fix the error, I suggest adding in the dependencies:
```
futures-util = { version = "0.3.1", default-features = false, features = ["sink"] }
```


## Your Environment
I reproduced this error on MacOS and linux.

* Rust Version (I.e, output of `rustc -V`): rustc 1.46.0 (04488afe3 2020-08-24) and rustc 1.48.0-nightly (397b390cc 2020-08-27)
* Actix Version: 0.10.0

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,7 +2,7 @@
 
 ## Unreleased - 2020-xx-xx
 ### Changed
-* Add `sink` feature for futures-util dependency. #427
+* Add `sink` feature for futures-util dependency. [#427]
 
 ## 0.10.0 - 2020-09-10
 ### Changed

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,7 +1,8 @@
 # CHANGES
 
 ## Unreleased - 2020-xx-xx
-
+### Changed
+* Add `sink` feature for futures-util dependency. #427
 
 ## 0.10.0 - 2020-09-10
 ### Changed

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,8 +1,11 @@
 # CHANGES
 
 ## Unreleased - 2020-xx-xx
-### Changed
-* Add `sink` feature for futures-util dependency. [#427]
+### Fixed
+* Fix error for build with --no-default-features flag, add `sink` feature for futures-util dependency. [#427]
+
+[#427]: https://github.com/actix/actix/pull/427
+
 
 ## 0.10.0 - 2020-09-10
 ### Changed

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -36,7 +36,7 @@ bytes = "0.5.3"
 crossbeam-channel = "0.4"
 derive_more = "0.99.2"
 futures-channel = { version = "0.3.1", default-features = false }
-futures-util = { version = "0.3.1", default-features = false }
+futures-util = { version = "0.3.1", default-features = false, features = ["sink"] }
 log = "0.4"
 pin-project = "0.4.17"
 once_cell = "1.4"

